### PR TITLE
doc: update quay enterprise ssl doc

### PR DESCRIPTION
**Pull request by kbrwn**
Updates method used to generate certs to match what is suggested for other products (tectonic).

- switch to using openssl.cnf file to generate certs
- new images with correct example url
- removed reference to lab.libcore.so

### CODE REVIEW

File: quay-enterprise/quay-ssl.md [hunk 1]
```diff
@@ -13,50 +13,45 @@ $ openssl genrsa -out rootCA.key 2048
...
 
-Sign the certificate with the CA:
-
-```
-$ openssl x509 -req -in ssl.csr -CA rootCA.pem -CAkey rootCA.key -CAcreateserial -out ssl.cert -days 500 -sha256
-Signature ok
-subject=/C=US/ST=California/L=SF/O=Demo Quay/OU=Demo Quay/CN=reg.example.com/emailAddress=support@quay.io
-Getting CA Private Key
-```
+The next step can be accomplished via the QE superuser panel or from the terminal:
```
Comment (robszumski) [inline quay-enterprise/quay-ssl.md]: should there be something after this colon?

**Comment by robszumski** [comment]
Text LGTM but I did not have time to test functionality

File: quay-enterprise/quay-ssl.md [hunk 2]
```diff
@@ -13,50 +13,45 @@ $ openssl genrsa -out rootCA.key 2048
...
+basicConstraints = CA:FALSE
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+subjectAltName = @alt_names
+[alt_names]
+DNS.1 = reg.example.com
+IP.1 = 12.345.678.9
 ```
 
-When creating the `ssl.csr` file it is important that the hostname of the server where QE is installed is used as the `Common Name` or QE will reject the  configuration. In this demo environment QE is currently installed at `reg.example.com`
+The following set of shell commands invoke the `openssl` utility to create a key for Quay Enterprise, generate a request for an Authority to sign a new certificate, and finally generate a certificate for the Quay Enterprise, signed by the CA created earlier.
```
Comment (joshix) [inline quay-enterprise/quay-ssl.md]: s/the Quay Enterprise/Quay Enterprise/

File: quay-enterprise/quay-ssl.md (hunk 1 above)
Comment (joshix) [inline quay-enterprise/quay-ssl.md]: s/:/./

File: quay-enterprise/quay-ssl.md [hunk 3]
```diff
@@ -103,11 +98,13 @@ $ docker restart cbe7b0fa39d8
...
 ### Test the secure connection
 
-Confirm the configuration by visiting the URL from a browser: `https://reg.example.com/`
+Confirm the configuration by visiting the URL from a browser `https://reg.example.com/`
 
 <img src="img/https-browser.png" class="img-center" alt="Browser"/>
 
-"Your Connection is not secure" means the CA is not officially and publicly trusted, but confirms that SSL is functioning properly. Check Google for how to configure your operating system and browser to trust a certificate signed by your own CA.
+
+"Your Connection is not secure" means the CA is untrusted but confirms that SSL is functioning properly. Google how to trust a CA based on the browser/operating system.
```
Comment (joshix) [inline quay-enterprise/quay-ssl.md]: Last sentence: Replace with:
"Check Google for how to configure your operating system and web browser to trust your new CA."

**Comment by joshix** [comment]
lgtm

**Pull request merged**